### PR TITLE
Support compilation mode

### PR DESCRIPTION
**Pull request by DamienCassou**

### CODE REVIEW

**Comment by DamienCassou** [comment]
any interest in this PR?

File: README.org [hunk 1]
```diff
@@ -34,4 +34,5 @@ manual intervention, use:
   (add-hook 'help-mode-hook #'epithet-rename-buffer)
   (add-hook 'occur-mode-hook #'epithet-rename-buffer)
   (add-hook 'shell-mode-hook #'epithet-rename-buffer)
+  (add-hook 'compilation-start-hook #'epithet-rename-buffer)
```
Comment (oantolin) [inline README.org]: Does this work? Or do you need to use the `epithet-rename-buffer-ignoring-arguments` function you defined?

Comment (DamienCassou) [reply README.org]: obviously it doesn't :-D. Thank you for catching that. It is fixed now.

**Comment by DamienCassou** [comment]
I think I took care of your feedback. Thank you.

**Pull request merged**